packages/block-editor-utils/src/types/index.d.ts

```diff
@@ -0,0 +1,4 @@
+declare global {
+  let wp: unknown;
```

Comment:
Global `wp` version is available only within the WordPress installation context.